templates/blank/package.json

```diff
@@ -10,6 +10,7 @@
     "devsafe": "rm -rf .next && cross-env NODE_OPTIONS=--no-deprecation next dev",
     "generate:importmap": "cross-env NODE_OPTIONS=--no-deprecation payload generate:importmap",
     "generate:types": "cross-env NODE_OPTIONS=--no-deprecation payload generate:types",
+    "preinstall": "sh -c '[ -f .env.example ] && mv .env.example .env || true'",
```

Comment:
This will fail `pnpm install` on Windows.
![image](https://github.com/user-attachments/assets/2fbda96e-4566-462c-a34a-50b255c00e15)